content/docs/config.md

```diff
@@ -149,36 +150,7 @@ not yet supported.
 </tab>
 </toggle>
 
-## CML Commands
-
-CML provides a number of commands to help package the outputs of ML workflows
-(including numeric data and visualizations about model performance) into a CML
-report.
-
-Below is a list of CML commands for starting cloud compute runners, writing and
-publishing Markdown reports to your CI/CD system.
-
-∞ **[`runner`](/doc/ref/runner)**\
-Launch a runner hosted by a cloud compute provider or locally on-premise (see [self-hosted runners](/doc/self-hosted-runners))\
-e.g. `cml runner launch --cloud={aws,azure,gcp,kubernetes} ...`
-
-∞ **[`pr`](/doc/ref/pr)**\
-Commit specified files to a new branch and create a pull request\
-e.g. `cml pr create "**/*.json" "**/*.py" --md >> report.md`
-
-∞ **[`comment`](/doc/ref/comment)**\
-Post a Markdown report as a commit comment\
-e.g. `cml comment create report.md`
-
-∞ **[`check`](/doc/ref/check)**\
-Post a Markdown report as a GitHub check\
-e.g. `cml check create report.md`
-
-∞ **[`tensorboard`](/doc/ref/tensorboard)**\
-Return a link to a <https://tensorboard.dev> page\
-e.g. `cml tensorboard connect --logdir=./logs --md >> report.md`
-
-### CML Reports
+<!-- ## CML Reports
 
 The `cml comment create` command can be used to post reports. CML reports are
 written in Markdown ([GitHub](https://github.github.com/gfm),
```

Comment:
Not sure what to do with this part. It seems to be covered already in all the GS versions? See all the `cml comment create` mentions in https://cml.dev/doc/start/github, for example.

But maybe it should be a separate page? And at this point we could group all these lose pages into a **User Guide** or **Manual** (or **Usage**) super section section. WDYT @casperdcl ?

Reply:
I figured out what to do here: moved all the CML report details to the `comment create` cmd ref. PTAL.